entries/jQuery.uniqueSort.xml

```diff
@@ -0,0 +1,45 @@
+<?xml version="1.0"?>
+<entry type="method" name="jQuery.uniqueSort" return="Array">
+  <title>jQuery.uniqueSort()</title>
+  <signature>
+    <added>3.0</added>
+    <argument name="array" type="Array">
+      <desc>The Array of DOM elements.</desc>
+    </argument>
+  </signature>
+  <desc>Sorts an array of DOM elements, in place, with the duplicates removed. Note that this only works on arrays of DOM elements, not strings or numbers.</desc>
+  <longdesc>
+    <p>The <code>$.uniqueSort()</code> function searches through an array of objects, sorting the array, and removing any duplicate nodes. A node is considered a duplicate if it is the <em>exact same</em> node as one already in the array; two different nodes with identical attributes are not considered to be duplicates. This function only works on plain JavaScript arrays of DOM elements, and is chiefly used internally by jQuery. You probably will never need to use it.</p>
+    <p>Prior to jQuery 3.0, this method was called <code><a href="/jQuery.unique/">jQuery.unique()</a></code>.</p>
```

Comment:
This is the only really new thing to this page. Rest is copied from the `jQuery.unique` entry.